v1/metrics/prometheus/config.go

```diff
@@ -0,0 +1,26 @@
+package prometheus
+
+const (
+	defaultFlushInterval = 5
```

Comment:
same here. `defaultFlushIntervalSec` maybe ?

Reply:
Changed in 5b3bab8.